src/services/models-manager.js

```diff
@@ -0,0 +1,26 @@
+const _ = require('lodash');
+
+module.exports = class ModelsManager {
+  constructor({ configStore }) {
+    this.configStore = configStore;
+    this.models = null;
+  }
+
+  getModels() {
+    if (!this.models) this._generateModelList();
+    return this.models;
+  }
+
+  getModelArray() {
+    return _.values(this.getModels());
+  }
+
+  _generateModelList() {
+    const models = this.configStore.Implementation.getModels();
```

Comment:
If I'm right, this line is used to be compliant with the current `forest-express` version, and the goal of the following PR is to remove `Implementation` here ?

Reply:
Yes exactly 👍